templates.json

```diff
@@ -114,6 +114,23 @@
       }
     ]
   },
+  {
+    "identifier": "product_configuration_link_extension",
+    "name": "Product configuration link",
+    "defaultName": "product-configuration-link",
+    "group": "Admin",
+    "supportLinks": [],
+    "url": "https://github.com/Shopify/extensions-templates",
+    "type": "product_configuration_link",
```

Comment:
Not 100% sure what the `type` here should be.

definition of `type` from https://vault.shopify.io/page/Create-an-extension-template~cqQS.md:

type: `identifier of the extension type that the template contains, it should be identical to the one defined in the Core specification`